pkg/controller/controller.go

```diff
@@ -522,3 +523,39 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
```

Comment:
Excellent, thanks a bunch for doing that.  This is exactly the kind of approach we need to take in this repo :100: